src/rpc/blockchain.cpp

```diff
@@ -207,6 +207,28 @@ UniValue getbestblockhash(const JSONRPCRequest& request)
     return chainActive.Tip()->GetBlockHash().GetHex();
 }
 
+UniValue getbestchainlock(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 0)
+        throw std::runtime_error(
+            "getbestchainlock\n"
+            "\nReturns the block hash of the best chainlock.\n"
+            "\nResult:\n"
+            "{\n"
+            "  \"blockhash\" : \"hash\",     (string) The block hash hex encoded\n"
+            "  \"height\" : n,          (numeric) The block height or index\n"
+            "}\n"
+            "\nExamples:\n"
+            + HelpExampleCli("getbestchainlock", "")
+            + HelpExampleRpc("getbestchainlock", "")
+        );
+    UniValue result(UniValue::VOBJ);
+    llmq::CChainLockSig clsig = llmq::chainLocksHandler->GetBestChainLock();
```

Comment:
I assume this will return a zero hash and -1 height when there is no chainlock yet? Is this ok as a return value or should we maybe throw an exception?

Reply:
Good point 👍 Would probably be better to throw an exception, just like for unknown blocks etc.